src/main/resources/templates/login2.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
Provide a more descriptive file name, and display the login type (management/user)

Reply:
Fixed